### ISSUE TITLE
[FEAT]: Link several containers at once?

### ISSUE THREAD

**Issue by tarbib** [azukaar/Cosmos-Server#79]
### Feature Description

Hello,
Not sure if I missed something it may be already possible...
Is it possible to link several containers at once?
It seems it's only 2 by 2 thus increasing the amount of networks needed to be created?

**Comment by azukaar** [comment]
Yes it is possible in the market for some entity as you mentioned the ticket is more about being able to do it directly in the servapp > network tab
While it is a good suggestion, it needs a bit of tweaking here and there because some logic in Cosmos do rely on those network to container <2 containers. But nothing that seems impossible
